### ISSUE TITLE
[BootRun] BootRrun Exception about Lombok

### ISSUE THREAD

**Issue by HUMBLEDDDD** [hezean/sustc#64]
### Related module

Code in sustc-api

### What happened

I tried to do bootrun, however there is this exception:
> Task :sustc-api:generateEffectiveLombokConfig FAILED

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':sustc-runner:generateEffectiveLombokConfig'.
> java.nio.charset.MalformedInputException: Input length = 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':sustc-api:generateEffectiveLombokConfig'.
> java.nio.charset.MalformedInputException: Input length = 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

**Comment by hezean** [comment]
did you succeed to run the program before? could you try adding this line to the end of `gradle.properties` in the root dir of this project?

```properties
org.gradle.jvmargs=-Dfile.encoding=UTF-8
```

if that still doesn't work, try adding these lines to `build.gradle.kts` under `sustc-runner` and `sustc-api` dir

```kotlin
tasks.withType<JavaCompile> {
    options.encoding = "UTF-8"
}
```

**Comment by HUMBLEDDDD** [comment]
Thanks for your response! No, I haven't run the program before. I tried adding the above lines to my code, and this time it stuck here and wouldn't continue：
```
22:36:14: Executing 'bootRun'...

> Task :sustc-runner:generateEffectiveLombokConfig UP-TO-DATE
> Task :sustc-api:generateEffectiveLombokConfig UP-TO-DATE
> Task :sustc-runner:processResources UP-TO-DATE
> Task :sustc-api:compileJava UP-TO-DATE
> Task :sustc-api:processResources NO-SOURCE
> Task :sustc-api:classes UP-TO-DATE
> Task :sustc-api:bootRunMainClassName UP-TO-DATE
> Task :sustc-api:bootRun SKIPPED
> Task :sustc-api:jar UP-TO-DATE
> Task :sustc-runner:compileJava UP-TO-DATE
> Task :sustc-runner:classes UP-TO-DATE
> Task :sustc-runner:bootRunMainClassName UP-TO-DATE

> Task :sustc-runner:bootRun

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::               (v2.7.16)

2023-12-23 22:36:16.463  INFO 676 --- [           main] io.sustc.Application                     : Starting Application using Java 17.0.4.1 on LAPTOP-05MN756V with PID 676 (D:\Desktop\TEST BOOK\database\project2\project2tmp\db_project2-main\sustc-runner\build\classes\java\main started by ���� in D:\Desktop\TEST BOOK\database\project2\project2tmp\db_project2-main\sustc-runner)
2023-12-23 22:36:16.463  INFO 676 --- [           main] io.sustc.Application                     : No active profile set, falling back to 1 default profile: "default"
2023-12-23 22:36:17.202  WARN 676 --- [           main] org.jline                                : Unable to create a system terminal, creating a dumb terminal (enable debug logging for more information)
2023-12-23 22:36:17.343  INFO 676 --- [           main] io.sustc.Application                     : Started Application in 1.199 seconds (JVM running for 1.481)
shell:>
```
exception is as follow:
```
Execution failed for task ':sustc-runner:bootRun'.
> Build cancelled while executing task ':sustc-runner:bootRun'

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
BUILD FAILED in 6m 24s
9 actionable tasks: 1 executed, 8 up-to-date
```

**Comment by hezean** [comment]
That's the shell itself, and you can enter commands to call your api

To get started, you can enter the "help" command

If you are intended to run the benchmark script, use the "benchmark" task as specified in the README file

**Comment by HUMBLEDDDD** [comment]
It works, thx a lot!

**Comment by Jaredanwolfgang** [comment]
HI! I have run into similar exceptions as you do. I tried all the methods above but still failed to bootRun my Gradle. I have runned the task this afternoon, I do not know what I have changed or trigger that has caused this errors...
<img width="959" alt="image" src="https://github.com/hezean/sustc/assets/83915005/9a79f286-d8dc-45d9-adb7-78447e834885">
When I tried to run with stacktrace it will show this: and I cannot understand this exception. What might possibly go wrong? 
<img width="587" alt="image" src="https://github.com/hezean/sustc/assets/83915005/2b739c55-895e-44ef-beb3-045f3c9cccd9">
(My repo should be the newest version, as I have just git merged them yesterday night ...)
